plugins/catalog/src/components/CatalogPage/CatalogLayout.tsx

```diff
@@ -18,29 +18,24 @@ import {
   configApiRef,
   Header,
   HomepageTimer,
-  identityApiRef,
   Page,
   useApi,
 } from '@backstage/core';
 import React from 'react';
-import { getTimeBasedGreeting } from './utils/timeUtil';
 
 type Props = {
   children?: React.ReactNode;
 };
 
 const CatalogLayout = ({ children }: Props) => {
-  const greeting = getTimeBasedGreeting();
-  const profile = useApi(identityApiRef).getProfile();
-  const userId = useApi(identityApiRef).getUserId();
-  const orgName = useApi(configApiRef).getOptionalString('organization.name');
+  const orgName =
+    useApi(configApiRef).getOptionalString('organization.name') ?? 'Backstage';
 
   return (
     <Page themeId="home">
       <Header
-        title={`${greeting.greeting}, ${profile.displayName || userId}!`}
-        subtitle={`${orgName || 'Backstage'} Service Catalog`}
-        tooltip={greeting.language}
+        title={`${orgName} Catalog`}
+        subtitle={`Index of software components in ${orgName}`}
```

Comment:
```suggestion
        subtitle={`Catalog of software components in ${orgName}`}
```

Catalog? (even if most in Europe will want catalogue ? :) )

Reply:
Yeah alright, a bit less technical perhaps :>